scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -419,7 +424,7 @@ object TreeSyntax {
             " (",
             t.cond,
             ") ",
-            p(Expr, t.thenp),
+            if (needParens) s("(", t.thenp, ")") else p(Expr, t.thenp),
```

Comment:
```suggestion
            w("(", p(Expr, t.thenp), ")", needParens),
```

Reply:
actually, maybe never mind, as `p()` could add own parens.

Reply:
if `p()` had a `force: Boolean` parameter we could use `p(Expr, t.thenp, force = needParens)`, maybe that would be better?

Reply:
> if `p()` had a `force: Boolean` parameter we could use `p(Expr, t.thenp, force = needParens)`, maybe that would be better?

that would be nice :)